common/lib/xmodule/xmodule/course_module.py

```diff
@@ -179,6 +179,12 @@ class CourseFields(object):
                  default=DEFAULT_START_DATE,
                  scope=Scope.settings)
     end = Date(help="Date that this class ends", scope=Scope.settings)
+    cosmetic_display_price = Integer(
+        display_name=_("Cosmetic Course Display Price"),
+        help=_("The cost displayed to students for enrolling in the course. If a paid course registration price is set by platform staff in the database, that price will be displayed instead of this one."),
+        default=0,
+        scope=Scope.settings,
+    )
```

Comment:
hrm. Also, I just realized that the currency symbol is coming from configuration and that is an instance wide singleton. So a single instance couldn't have display prices that are mixed currency. Again, this is a general limitation of the shoppingcart, but I figured I'd just mention it.